test/test_resize.cpp

```diff
@@ -1631,6 +1631,37 @@ TEST_F(ResizeTest, FusionResizePadWithValue) {
   NVF_CHECK(ref.equal(cg_outputs[0]));
 }
 
+// Same as Pad1 but pad by negative value to create an empty tensor
+TEST_F(ResizeTest, FusionResizePadToEmptyTensor) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  std::vector<int64_t> shape({4, 2});
+
+  auto tv0 = makeSymbolicTensor(2);
+  fusion->addInput(tv0);
+
+  auto tv1 =
+      pad(tv0,
+          {IrBuilder::create<Val>(-1L), IrBuilder::create<Val>(-1L)},
+          IrBuilder::create<Val>(2.0));
+  fusion->addOutput(tv1);
+  // set allocation domain to trigger validation check on size/stride
+  tv1->setAllocationDomain(tv1->getMaybeRFactorDomain(), true);
```

Comment:
setting the allocation order triggers the validation.

Technically we can try to patch the contiguous_stride computation by `std::max(size, 1)`. But why do we even bother validating a size-0 tensor, if we are not going to index into it at all.
https://github.com/NVIDIA/Fuser/blob/1c2a4ff4924f4d187ab290210ab161013bd4fcf8/csrc/tensor_metadata.cpp#L249